.cloudbuild/ci/integration-tests.yaml

```diff
@@ -1,29 +1,69 @@
+timeout: 20m
+options:
+  machineType: E2_HIGHCPU_32
+  
+  volumes:
+    # A place to put shared state between build steps so that we don't pollute
+    # the workspace
+    - name: state
+      path: '/state'
```

Comment:
Hrm, how does this `/state` directory work? Can that submits a PR write to it?

Reply:
It's an ephemeral volume that gets mounted in every step and discarded after the build

Reply:
Let's talk about this tomorrow, have to think about this some more.